lms/djangoapps/teams/static/teams/templates/edit-team.underscore

```diff
@@ -0,0 +1,33 @@
+<div class="create-team wrapper-msg is-incontext urgency-low warning is-hidden">
+    <div class="msg">
+        <div class="msg-content">
+            <h3 class="title"><%- gettext("Oops!") %></h3>
```

Comment:
@muhammad-ammar I think my comments from a particular commit got lost. My suggestion was to do without an error title here, i.e. remove the "Oops!" title.